src/java/org/apache/cassandra/repair/autorepair/RepairTokenRangeSplitter.java

```diff
@@ -109,8 +110,9 @@
  *         repaired.
  *     </li>
  *     <li>
- *         <b>partitions_per_assignment</b>: The target number of partitions that should be included in a repair
- *         assignment.  This configuration exists to reduce excessive overstreaming.
+ *         <b>partitions_per_assignment</b>: The maximum number of partitions that should be included in a repair
+ *         assignment.  This configuration exists to reduce excessive overstreaming by attempting to limit the number
```

Comment:
Nit: There are two spaces. 
`s/assignment.  This/assignment. This/`

Reply:
also see it in a few other places in this file, will fix it.

Reply:
fixed :+1: